eth/tracers/native/prestate.go

```diff
@@ -120,6 +120,7 @@ func (t *prestateTracer) CaptureStart(env *vm.EVM, from libcommon.Address, to li
 		// The sender balance is after reducing: value.
 		// We need to re-add it to get the pre-tx balance.
 		fromBal.Add(fromBal, valueBig)
+		fromBal.Add(fromBal, env.BlobFee.ToBig())
```

Comment:
Better move this line next to `fromBal.Add(fromBal, consumedGas)` to highlight the similarity between regular and blob gas.

Reply:
Thought about that, blob txs can't create tho, some teeny bit savings.

Reply:
Well, I'd prefer clarity over the tiny saving here, but OK.